android/app/src/main/java/poke/rogue/helper/presentation/ability/AbilityActivity.kt

```diff
@@ -2,65 +2,25 @@ package poke.rogue.helper.presentation.ability
 
 import android.content.Context
 import android.content.Intent
-import android.net.Uri
 import android.os.Bundle
-import android.view.Menu
-import android.view.MenuItem
+import androidx.appcompat.widget.Toolbar
 import androidx.fragment.app.commit
 import androidx.fragment.app.replace
 import poke.rogue.helper.R
 import poke.rogue.helper.databinding.ActivityAbilityBinding
 import poke.rogue.helper.presentation.base.BindingActivity
-import poke.rogue.helper.presentation.util.context.drawableOf
-import poke.rogue.helper.presentation.util.context.stringOf
-import poke.rogue.helper.presentation.util.context.toast
 
 class AbilityActivity : BindingActivity<ActivityAbilityBinding>(R.layout.activity_ability) {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-
-        initViews()
         if (savedInstanceState == null) {
             supportFragmentManager.commit {
                 replace<AbilityFragment>(R.id.fragment_container_ability)
             }
         }
     }
 
-    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
-        menuInflater.inflate(R.menu.menu_toolbar, menu)
-        return super.onCreateOptionsMenu(menu)
-    }
-
-    override fun onOptionsItemSelected(item: MenuItem): Boolean {
-        when (item.itemId) {
-            R.id.item_toolbar_pokerogue -> {
-                navigateToPokeRogue()
-            }
-
-            R.id.item_toolbar_feedback -> {
-                toast(R.string.toolbar_feedback)
-            }
-
-            android.R.id.home -> {
-                onBackPressedDispatcher.onBackPressed()
-            }
-        }
-        return true
-    }
-
-    private fun initViews() =
-        with(binding) {
-            setSupportActionBar(toolbarAbility.toolbar)
-            toolbarAbility.toolbar.overflowIcon = drawableOf(R.drawable.ic_menu)
-            supportActionBar?.setDisplayShowTitleEnabled(false)
-        }
-
-    private fun navigateToPokeRogue() {
-        val intent =
-            Intent(Intent.ACTION_VIEW, Uri.parse(stringOf(R.string.home_pokerogue_url)))
-        startActivity(intent)
-    }
+    override fun toolBar(): Toolbar? = null
```

Comment:
Activity 가 무언가를 반환하는 것이 어색한 것 같습니다.
그리고, 네이밍상 toolbar 를 만드는 함수이지만 실제로는 [템플릿 메서드 패턴](https://refactoring.guru/ko/design-patterns/template-method)으로 사용되고 있네요  

다음과 같이 제안드립니다.

```suggestion
// ToolBarActivity 
protected var toolbar: Toolbar? = null


// SubActivity
override val toolbar get() = binding.toolbarType
```